### ISSUE TITLE
Add config option for Facebook Pixel ID

### ISSUE THREAD

**Issue by pestevez** [facebook/Docusaurus#503]
### Is this a bug report?

No, this is a feature request.

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes.

### Proposal

Add a new config field `facebookPixelId` to support analytics using the [Facebook Pixel](https://www.facebook.com/business/a/facebook-pixel).

We would then add the following script in Site.js:

```javascript
<script>
     !function(f,b,e,v,n,t,s)
     {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
     n.callMethod.apply(n,arguments):n.queue.push(arguments)};
     if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
     n.queue=[];t=b.createElement(e);t.async=!0;
     t.src=v;s=b.getElementsByTagName(e)[0];
     s.parentNode.insertBefore(t,s)}(window, document,'script',
     'https://connect.facebook.net/en_US/fbevents.js');
     fbq('init', 'YOUR-PIXEL-ID');
     fbq('track', 'PageView');
</script>
```

similar to what we do for the Facebook SDK.

**Comment by pestevez** [comment]
I am happy to submit a pull request if this proposal makes sense to the team.

**Comment by JoelMarcey** [comment]
@pestevez Hi! Nice to virtually meet a fellower FB'er. I would love for you to submit a PR for this. I don't see why we wouldn't want something like this, but a PR would help us get more eyes on it possibly.

I am going to consider this issue claimed.

**Comment by pestevez** [comment]
Nice to e-meet you too, @JoelMarcey! The PR is coming soon.

**Comment by pestevez** [comment]
Fixed by #507
